napalm/ios/ios.py

```diff
@@ -2173,6 +2173,52 @@ def get_config(self, retrieve='all'):
 
         return configs
 
+    def get_ipv6_neighbors_table(self):
+        """
+        Get IPv6 neighbors table information.
+        Return a list of dictionaries having the following set of keys:
+            * interface (string)
+            * mac (string)
+            * ip (string)
+            * age (float) in seconds
+            * state (string)
+        For example::
+            [
+                {
+                    'interface' : 'MgmtEth0/RSP0/CPU0/0',
+                    'mac'       : '5c:5e:ab:da:3c:f0',
+                    'ip'        : '2001:db8:1:1::1',
+                    'age'       : 1454496274.84,
+                    'state'     : 'REACH'
+                },
+                {
+                    'interface': 'MgmtEth0/RSP0/CPU0/0',
+                    'mac'       : '66:0e:94:96:e0:ff',
+                    'ip'        : '2001:db8:1:1::2',
+                    'age'       : 1435641582.49,
+                    'state'     : 'STALE'
+                }
+            ]
+        """
+
+        ipv6_neighbors_table = []
+        command = 'show ipv6 neighbors'
+        output = self._send_command(command)
+
+        for entry in output.split('\n')[4:]:
+            # typical format of an entry in the IOS IPv6 neighbors table:
+            # 2002:FFFF:233::1 0 2894.0fed.be30  REACH Fa3/1/2.233
+            ip, age, mac, state, interface = entry.split()
+            mac = -1 if mac == '-' else napalm.base.helpers.mac(mac)
```

Comment:
mac will have to be a string for the unknown value and not an integer (or else it will fail the type check for the value). So this should be set to a null string.